TradeItIosTicketSDK2/TradeItConnector.swift

```diff
@@ -32,6 +32,16 @@ internal extension TradeItConnector {
         _ request: URLRequest,
         withCompletionBlock completionBlock: @escaping (TradeItResult, String?) -> Void
     ) {
+        guard !TradeItSDK.isDeviceJailbroken else {
+            completionBlock(
+                TradeItErrorResult(
```

Comment:
@TheAntonioReyes , @jsom This will return a TradeItErroResult for any request if the device is jailbroken.

Reply:
@TheAntonioReyes will this satisfy the pen test issue?

Reply:
take out the space and bang " !" and we should be good